src/cider/nrepl/middleware/test/extensions.clj

```diff
@@ -0,0 +1,35 @@
+(ns cider.nrepl.middleware.test.extensions
+  "Extensions to `clojure.test` functionality.
+
+  These are kept in a separate namespace because they are, by definition,
+  opinionated."
+  (:require [clojure.data :as data]
+            [clojure.pprint :as pp]
+            [clojure.test :as test :refer [assert-expr]]))
+
+;; From pjstadig/humane-test-output
+;; https://github.com/pjstadig/humane-test-output
+(defmethod assert-expr '= [msg [_ expected & more]]
```

Comment:
In cases where an `is` form is part of a macro expansion `assert-expr` will get called with the fully qualified name for `=` (`clojure.core/=`), so in humane-test-output I had to have a `defmethod` for that as well. If there isn't some other way to work around it, then you may want to do the same here.

Reply:
Hmm, alright. Seems to me `assert-expr` should dispatch on the Var, not the symbol, but that's outside of our control.